client/app/index.html

```diff
@@ -221,6 +222,8 @@ <h3 translate>About</h3>
             'liveblog.posts',
             'liveblog.blog',
             'liveblog.themes',
+            'liveblog.freetypes',
```

Comment:
In this case you could create submodules like: `liveblog.freetype.crud` and `liveblog.freetype.manip` or whatever you want to call it. Because this is confusing